ccl/changefeedccl: unit tests fail if no network acces

The following tests fail if run in an environment with no network access:

* `TestCreateChangefeedScheduleInExplicitTxnRollback`
* `TestCreateChangefeedScheduleIfNotExists`

```
--- FAIL: TestCreateChangefeedScheduleIfNotExists (134.79s)
    test_log_scope.go:167: test logs captured to: outputs.zip/logTestCreateChangefeedScheduleIfNotExists4155146104
    test_log_scope.go:81: use -show-logs to present logs inline
    scheduled_changefeed_test.go:348: error executing 'CREATE SCHEDULE IF NOT EXISTS 'foo' FOR CHANGEFEED TABLE t1 INTO 's3://bucket?AUTH=implicit' WITH initial_scan = 'only' RECURRING '@daily'': pq: Failed to dry run create changefeed: could not find s3 bucket's region: RequestError: send request failed
        caused by: dial tcp: lookup s3.amazonaws.com on 169.254.169.254:53: dial udp 169.254.169.254:53: connect: network is unreachable
    panic.go:522: -- test log scope end --
FAIL
```

Epic CRDB-8308

Jira issue: CRDB-30672

--- a/pkg/ccl/changefeedccl/scheduled_changefeed_test.go
+++ b/pkg/ccl/changefeedccl/scheduled_changefeed_test.go
@@ -343,7 +343,7 @@ func TestCreateChangefeedScheduleIfNotExists(t *testing.T) {
 	th.sqlDB.Exec(t, "CREATE TABLE t1 (a INT)")
 
 	const scheduleLabel = "foo"
-	const createQuery = "CREATE SCHEDULE IF NOT EXISTS '%s' FOR CHANGEFEED TABLE t1 INTO 's3://bucket?AUTH=implicit' WITH initial_scan = 'only' RECURRING '@daily'"
+	const createQuery = "CREATE SCHEDULE IF NOT EXISTS '%s' FOR CHANGEFEED TABLE t1 INTO 'null://' WITH initial_scan = 'only' RECURRING '@daily'"
 
 	th.sqlDB.Exec(t, fmt.Sprintf(createQuery, scheduleLabel))
 
@@ -387,7 +387,7 @@ func TestCreateChangefeedScheduleInExplicitTxnRollback(t *testing.T) {
 	require.NoError(t, res.Err())
 
 	th.sqlDB.Exec(t, "BEGIN;")
-	th.sqlDB.Exec(t, "CREATE SCHEDULE FOR CHANGEFEED TABLE t1 INTO 's3://bucket?AUTH=implicit' WITH initial_scan = 'only' RECURRING '@daily';")
+	th.sqlDB.Exec(t, "CREATE SCHEDULE FOR CHANGEFEED TABLE t1 INTO 'null://' WITH initial_scan = 'only' RECURRING '@daily';")
 	th.sqlDB.Exec(t, "ROLLBACK;")
 
 	res = th.sqlDB.Query(t, "SELECT id FROM [SHOW SCHEDULES FOR CHANGEFEED]")

